### ISSUE TITLE
AWS_NO2-VISUALISATION layer transparent for last 3 dates

### ISSUE THREAD

**Issue by lubojr** [eurodatacube/eodash#2642]
Following new dates of layer AWS_NO2-VISUALISATION show transparent map: 2024-06-03, 2024-06-10, 2024-06-17

**Comment by AlessandroScremin** [comment]
@lubojr 
the images have been reingestedd. Now should work correctly

check and close the ticket

**Comment by santilland** [comment]
@AlessandroScremin the days specified in the initial issue are fixed, but it seems that the follow on dates also only show empty data, so 2024-07-08, 2024-07-15

**Comment by AlessandroScremin** [comment]
@santilland 
Again i reingested the datasets and now is working again. i will investigate the issue, but Im wondering if this is linked to the issue I got with hub.eox.at workspace using the headless execution. Honestly I simply re used the same script but from the workspace intead of headless. And this time is working.

What is strange is that I run the same script adn I got values images but with all pixels values to 0. then executed the same identical script and now the images are ok.

I need to investigate why I have ingested images with no failures, but the data is not processed correctly.
